tests/aws/services/kms/test_kms.py

```diff
@@ -585,7 +587,12 @@ def test_import_key(self, kms_create_key, aws_client, snapshot):
 
         # import symmetric key (key material) into KMS
         public_key = load_der_public_key(params["PublicKey"])
-        encrypted_key = public_key.encrypt(symmetric_key, PKCS1v15())
+        encrypted_key = public_key.encrypt(
+            symmetric_key,
+            padding.OAEP(
+                mgf=padding.MGF1(algorithm=hashes.SHA256()), algorithm=hashes.SHA256(), label=None
+            ),
```

Comment:
we can directly import the class `OAEP` as referred [here](https://cryptography.io/en/2.7/_modules/cryptography/hazmat/primitives/asymmetric/padding/#:~:text=(AsymmetricPadding)-,class%20OAEP(object)%3A,-name%20%3D%20%22EME) and skip passing `label` as default value for it is already `None`.

Reply:
Doing this leads to a test failure:
```
E       TypeError: OAEP.__init__() missing 1 required positional argument: 'label'

tests/aws/services/kms/test_kms.py:592: TypeError
```
If I check the current code of the library (2.7 is quite old) here, none of the arguments have defaults: https://github.com/pyca/cryptography/blob/main/src/cryptography/hazmat/primitives/asymmetric/padding.py#L67, so for now we need to pass `label` explicitly.